src/stories/Tabs.stories.tsx

```diff
@@ -30,35 +30,23 @@ export const Default: StoryFn<TabsProps> = (args) => {
     return (
         <div style={{ width: args.layout === 'vertical' ? '300px' : '500px', height: '200px' }}>
             <Tabs {...args}>
-                <Tab name="tab one" label={<TabLabel label="First" />} selected>
-                    <TabContentWrapper>
-                        <TabContent>content for first</TabContent>
-                    </TabContentWrapper>
+                <Tab name="1" label={<TabLabel label="First" />} selected>
```

Comment:
Why we need `selected` prop here if you add `active` to Tabs component?

Reply:
I forgot to drop this prop